tests/v2/03_api/test_api_task_collection_failures.py

```diff
@@ -133,3 +134,35 @@ async def test_missing_task_executable(
         assert data["status"] == "fail"
         assert data["log"]  # This is because of verbose=True
         assert "fail" in data["log"]
+
+
+async def test_collection_validation_error(
+    client,
+    MockCurrentUser,
+    override_settings_factory,
+    tmp_path: Path,
+    testdata_path: Path,
+):
+    override_settings_factory(FRACTAL_TASKS_DIR=tmp_path)
+
+    file_dir = tmp_path / ".fractal/fractal-tasks-mock0.0.1"
+    file_path = file_dir / "collection.json"
+    os.makedirs(file_dir, exist_ok=True)
+    with open(file_path, "w") as f:
```

Comment:
We could add another test here, before writing the JSON file.
The response status is still 422, but due to a different exception (`FileNotFoundError`)

Reply:
Addressed with https://github.com/fractal-analytics-platform/fractal-server/pull/1379/commits/5cf28ed491d6baeb0b767bd70d619f2f6aea2199